WebJobs.Script.sln

```diff
@@ -37,6 +37,7 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{6D78685E-D299-4803-9923-12143F0CA41F}"
 	ProjectSection(SolutionItems) = preProject
 		build\common.props = build\common.props
+		build\package.props = build\package.props
```

Comment:
I split this out b/c I wanted to share the package metadata with the generator package, but didn't want a lot of the other stuff. The references to the CodeAnalysis packages were causing some issues and unnecessary for that project.